src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,6 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    abstract public boolean isMutating();
```

Comment:
Good job in making this abstract class. 

This method should have a header comment. If not, users of the method as well as those who overrides method will not know how exactly the method behaves.